amphp/artax/2017-05-09.yaml

```diff
@@ -0,0 +1,11 @@
+title:     Cookie leakage to wrong origins and non-restricted cookie acceptance
+link:      https://twitter.com/asyncphp/status/862033196824842246
+cve:       ~
+branches:
+    master:
+        time:     2017-05-09 19:42:02
+        versions: ['>=2', '<2.0.6']
+    1.x:
+        time:     2017-05-09 19:42:02
+        versions: ['>=1', '<1.0.6']
```

Comment:
Changed. How are branch names handled? Do they actually matter? Because `master` will change to `2.x` soon.